app/editor/src/features/content/morning-report/MorningReport.tsx

```diff
@@ -48,15 +42,15 @@ export const MorningReport: React.FC<IMorningReportProps> = (props) => {
   );
 
   const fetch = React.useCallback(
-    async (filter: IMorningReportFilter) => {
+    async (filter: IMorningReportFilter & Partial<IContentListAdvancedFilter>) => {
       try {
         setLoading(true);
         const data = await findContent(
           makeFilter({
             ...filter,
           }),
         );
-        return new Page(data.page - 1, data.quantity, data?.items, data.total);
+        return new Page(data.page - 1, data.quantity, [], data.total);
```

Comment:
How come you make a request for content and then return an empty array?

Reply:
This was a mistake from testing! Reverting